FE/src/components/CommentCard.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import NavButton from './NavButton';
+
+export default function ReviewInfo ({token, comment}) {
```

Comment:
This needs to be said about a lot of components not just these, but if we don't actually use the token, it doesn't need to be passed in.